Dockerfile

```diff
@@ -1,22 +1,41 @@
 # Base will install runtime dependencies and configure generics
-FROM node:12.16
+FROM node:12-slim as base
+LABEL maintainer="Marko Kajzer <markokajzer91@gmail.com>, Nico Stapelbroek <discord-soundbot@nstapelbroek.com>"
+
+RUN apt-get update -q && apt-get install -qy wget
+RUN wget -qO /tini https://github.com/krallin/tini/releases/download/v0.18.0/tini-$(dpkg --print-architecture) && \
+    chmod +x /tini && mkdir /app && chown -R node:node /app
```

Comment:
Do we need the mkdir part here? My understanding was that the `WORKDIR /app` will take part of that

Reply:
that's true. My problem at that time was that  WORKDIR creates the directory without chowning it to a user.

I see this doesn't make a lot of sense now. I'll switch it around to WORKDIR -> RUN (chown).

Reply:
Followup question. Is it possible to achieve the same with

```docker
USER node
WORKDIR /app
```